R/counts.R

```diff
@@ -112,7 +112,7 @@ get_single_cell_experiment <- function(
         has_name(raw_data, c("cell_", "file_id_db"))
     )
 
-    versioned_cache_directory <- file.path(cache_directory, COUNTS_VERSION)
```

Comment:
This variable isn't needed at all anymore, you can just use `cache_directory` directly.